src/Execution/Resolver/PropertyFieldResolver.php

```diff
@@ -1,36 +0,0 @@
-<?php
-
-namespace Digia\GraphQL\Execution\Resolver;
-
-use Digia\GraphQL\Error\ExecutionException;
-use Digia\GraphQL\Execution\ExecutionEnvironment;
-
-class PropertyFieldResolver implements ResolverInterface
```

Comment:
I would like to keep this class based if possible because we will need to add resolver classes soon.

Reply:
You could wire this up through DI instead of using a method.